apps/website-25/src/components/courses/SocialShare.tsx

```diff
@@ -9,31 +9,32 @@ type SocialShareProps = {
 
 const SocialShare: React.FC<SocialShareProps> = ({ coursePath, referralCode, text }) => {
   const constructFullCourseUrl = (campaign: string) => {
-    const url = `bluedot.org${coursePath}?utm_source=referral&utm_campaign=${campaign}`;
+    const baseUrl = window.location.origin;
+    const url = `${baseUrl}${coursePath}?utm_source=referral&utm_campaign=${campaign}`;
```

Comment:
I changed this to use the path from the window to make it easier to test locally